content/en/ecosystem/distributions.md

```diff
@@ -20,7 +20,7 @@ component they customize.
 endorse** the third-party distributions listed in the following table. The list
 is provided as a convenience for the community. {{% /alert %}}
 
-{{% ecosystem/distributions-table %}}
+{{% ecosystem/distributions-table filter="non-collector" %}}
```

Comment:
Shouldn't this display all distributions as it did before? 
Otherwise, how does a reader know to look elsewhere for Collector distributions?

Reply:
At a minimum there should be a Note (alert), that lets the reader know where to find OTel (first-party) Collector distros, by linking to the (now updated) Collector distro page.